Formula/git-annex.rb

```diff
@@ -3,6 +3,7 @@ class GitAnnex < Formula
   homepage "https://git-annex.branchable.com/"
   url "https://hackage.haskell.org/package/git-annex-8.20200720.1/git-annex-8.20200720.1.tar.gz"
   sha256 "d6252697151ab01cc8f7a492dc5eb4b4e773c0d48771df360c82a7cca7a39c3c"
+  license "AGPL-3.0-only"
```

Comment:
```suggestion
  license "AGPL-3.0-or-later"
```

https://git-annex.branchable.com/license/